Week1/homework/queries_db_tb.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const dropDbWorld = `DROP SCHEMA IF EXISTS world`;
+const creatDbWorld = `CREATE SCHEMA world`;
+
+const dropTableCountry = `DROP TABLE IF EXISTS country`;
+const CreateTableCountry = `CREATE TABLE  country  (
+  Code  CHAR(3) NOT NULL DEFAULT '',
+  Name  VARCHAR(52) NOT NULL DEFAULT '',
+  Continent  enum('Asia','Europe','North America','Africa','Oceania','Antarctica','South America') NOT NULL DEFAULT 'Asia',
+  Region  VARCHAR(26) NOT NULL DEFAULT '',
+  SurfaceArea  FLOAT(10,2) NOT NULL DEFAULT '0.00',
+  IndepYear  SMALLINT(6) ,
+  Population  INT(11) NOT NULL DEFAULT '0',
+  LifeExpectancy  FLOAT(3,1) ,
+  GNP  FLOAT(10,2) ,
+  GNPOld  FLOAT(10,2) ,
+  LocalName  VARCHAR(45) NOT NULL DEFAULT '',
+  GovernmentForm  VARCHAR(45) NOT NULL DEFAULT '',
+  HeadOfState  VARCHAR(60) ,
+  Capital  VARCHAR(22) NOT NULL DEFAULT ''
+) `;
+
+const dropTableCity = `DROP table IF EXISTS city`;
+const CreateTableCity = `CREATE TABLE city (
+  ID INT(11) NOT NULL AUTO_INCREMENT,
+  Name VARCHAR(35) NOT NULL DEFAULT '',
+  CountryCode CHAR(3) NOT NULL DEFAULT '',
+  District VARCHAR(20) NOT NULL DEFAULT '',
+  Population INT(11) NOT NULL DEFAULT '0',
+  PRIMARY KEY (ID)
+)`;
+
+const insertContires = [
+    "INSERT INTO country VALUES ('USA','United States','North America','North America',9363520.00,1776,278357000,77.1,8510700.00,8110900.00,'United States','Federal Republic','George W. Bush','WASHINGTON, D.C.')",
```

Comment:
Good also !

Reply:
If you want to go a bit further you could try to do only ONE INSERT query but entering multiple rows. It helps reducing the number of calls on the DB.

Reply:
I am going to read about that.. interesting.

Reply:
I leave also a reference for me, 
https://www.w3schools.com/nodejs/nodejs_mysql_insert.asp